frontend/src/modules/automation/pages/automation-list-page.vue

```diff
@@ -1,154 +1,187 @@
 <template>
-  <div class="pt-4">
-    <div
-      v-if="loading('table') && count === 0"
-      v-loading="loading('table')"
-      class="app-page-spinner"
-    />
-    <div v-else-if="count > 0">
-      <div
-        class="flex items-center py-1 mb-3 mt-2"
-        :class="count ? 'justify-between' : 'justify-end'"
-      >
-        <div class="text-gray-500 text-sm">
-          {{ pluralize('webhook', count, true) }}
-        </div>
-        <div>
-          <el-button
-            class="btn btn--primary btn--sm !h-8"
-            @click="onAddWebhookClick"
+  <div class="pt-6">
+    <div class="flex justify-between pb-6">
+      <div class="flex items-center">
+        <div class="flex text-xs text-gray-600">
+          <div
+            v-for="option in options"
+            :key="option.value"
+            class="h-8 border-solid border-gray-200 border-r border-y first:border-l flex items-center
+       justify-center transition hover:bg-gray-50 cursor-pointer first:rounded-l-md last:rounded-r-md px-3 bg"
+            :class="filter.type === option.value ? 'font-medium bg-gray-100 text-gray-900' : 'bg-white'"
+            @click="changeAutomationFilter({ type: option.value })"
           >
-            Add webhook
-          </el-button>
+            {{ option.label }}
+          </div>
         </div>
+        <p class="pl-4 text-xs leading-5 text-gray-500">
+          {{ pluralize('automation', totalAutomations, true) }}
+        </p>
+      </div>
+      <div>
+        <el-popover
+          trigger="click"
+          placement="bottom-end"
+          width="15rem"
+          popper-class="!p-2"
+        >
+          <template #reference>
+            <el-button class="btn btn--primary btn--sm flex items-center !py-2">
+              Add automation
+              <span class="ri-arrow-down-s-line text-base ml-2 flex items-center h-4" />
+            </el-button>
+          </template>
+          <div class="popover-item h-auto mb-1 py-2 px-2.5" @click="createAutomation('webhook')">
+            <div class="flex">
+              <div class="mt-0.5">
+                <img alt="Webhook" src="/images/automation/webhook.png" class="w-4 max-w-4">
+              </div>
+              <div class="pl-2">
+                <h6 class="text-xs leading-5 font-medium mb-0.5 text-gray-900">
+                  Webhook
+                </h6>
+                <p class="text-2xs leading-4.5 text-gray-500 break-words">
+                  Send webhook payloads to automate workflows
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="popover-item h-auto py-2 px-2.5" @click="authenticateSlack">
```

Comment:
I believe the `authenticateSlack` should be the click handler on the `Install app` button and the dropdown option should just open the drawer